spec/features/admin/add_custom_organisation_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'support/confirmation_use_case'
+
+describe 'adding a custom organisation name' do
+  include_examples 'confirmation use case spy'
+
+  let!(:admin_user) { create(:user, super_admin: true) }
+
+  context 'when visiting the custom organisations page' do
+    before do
+      sign_in_user admin_user
+      visit admin_custom_organisations_path
+    end
+
+    it 'will show the add custom organisations page' do
+      expect(page.body).to have_content('Allow an organisation access to the admin platform')
+    end
+
+    it 'will redirect you to organiastions page once a custom org is added' do
+      fill_in "Enter the organisation's full name", with: 'Custom Org name'
+      click_on 'Allow organisation'
```

Comment:
I would check that the `CustomOrganisationName` has actually been created here. You would do that like so:
```
expect do
  click_on 'Allow organisation'
end.to change {CustomOrganisationName.count}.by(1)
```
There is some debate about checking the database like this from feature specs, but as this is the only spec checking the create action (theres no controller or request specs, which makes sense), then I think we should cover it here.